codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -357,6 +357,8 @@ sterilised->sterilized
 steriliser->sterilizer
 sterilises->sterilizes
 sterilising->sterilizing
+storey->story
```

Comment:
Move to _rare_ (synonym of floor), same for plural.

Reply:
Note that this is the `dictionary_en-GB_to_en-US.txt` file: https://en.wikipedia.org/wiki/Storey#cite_note-Cambridge_University_Press_-_Cambridge_Dictionary_-_storey-1

So I think it looks correct?